Formula/scheme48.rb

```diff
@@ -21,4 +21,19 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    (testpath/"hello.scm").write <<-EOS.undent
+      (display "Hello, World!") (newline)
+    EOS
+
+    expected = <<-EOS.undent
+      Hello, World!\#{Unspecific}
+
+      \#{Unspecific}
```

Comment:
This particular interpreter doesn't have a file input argument, so it has to be fed lines of Scheme one-by-one in batch mode. AFAICT, the superfluous newlines and `#{Unspecifc}`s are the product of that one-by-one interpretation.

The other option is to generate an image file and load the interpreter with that, but that's a 3-4 step process compares to the two steps here (write file, execute).